packages/did-ethr-resolver/src/implementations/operator.ts

```diff
@@ -137,8 +149,8 @@ export class Operator extends Resolver implements IOperator {
     did: string,
     didAttribute: DIDAttribute,
     updateData: IUpdateData,
-    validity: number = Number.MAX_SAFE_INTEGER,
-  ): Promise<utils.BigNumber> {
+    validity: number = Number.MAX_SAFE_INTEGER - 1,
```

Comment:
Why the `-1` here? Maybe a comment would be appropriate

Reply:
With ethers v5, the instruction `BigNumber.from` throws the following error: 
`Error: overflow (fault="overflow", operation="BigNumber.from", value=9007199254740991, code=NUMERIC_FAULT, version=bignumber/5.4.0)`.

Adding `-1` to MAX_SAFE_INTEGER fixes this.